src/server/logging/log_interceptor.js

```diff
@@ -70,13 +94,23 @@ export class LogInterceptor extends Stream.Transform {
    *  @param {object} - log event
    */
   downgradeIfEcanceled(event) {
-    return downgradeIfErrorMatches('ECANCELED', event);
+    return downgradeIfErrorType('ECANCELED', event);
+  }
+
+  downgradeIfHTTPSWhenHTTP(event) {
+    return downgradeIfErrorMessage('Parse Error', event);
```

Comment:
Good point, I went straight for text because there was no errno.  But there is a code for Parse  Error sot hat looks like it's the way to go:
```
{
    event: 'log',
    timestamp: 1492194353021,
    tags: ['connection', 'client', 'error'],
    data: {
        Error: Parse Error at Error(native) bytesParsed: 0,
        code: 'HPE_INVALID_METHOD'
    },
    pid: 73405
}
```

For the http when https case there isn't more:
```
{
    event: 'log',
    timestamp: 1492194569892,
    tags: ['connection', 'client', 'error'],
    data: Error: 140736014037952: error: 1407609 C: SSL routines: SSL23_GET_CLIENT_HELLO: http request: ../deps/openssl/openssl/ssl/s23_srvr.c:394:at Error(native),
    pid: 74537
}
```